test/v200/apiTest/api_test.go

```diff
@@ -0,0 +1,80 @@
+package apiTest
+
+import (
+	"testing"
+
+	schema "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+	api "github.com/devfile/api/v2/test/v200/utils/api"
+	common "github.com/devfile/api/v2/test/v200/utils/common"
```

Comment:
no need to alias, already is api and common

Reply:
I plan in my next pull request to make these commonUtils and apiUtils.

Reply:
Fixed in new commit